addons/sale_stock/models/stock_config_settings.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class StockConfigSettings(models.TransientModel):
+    _inherit = 'stock.config.settings'
+
+    security_lead = fields.Float(related='company_id.security_lead', default=lambda self: self.env.user.company_id.security_lead)
```

Comment:
Related field with default value doesn't make any sense. If the value is not correctly set, it's probably because the field company_id isn't displayed on the form.